tests/src/test/scala/cats/tests/RegressionTests.scala

```diff
@@ -85,7 +85,7 @@ class RegressionTests extends CatsSuite {
 
   test("#500: foldMap - traverse consistency") {
     assert(
-      List(1,2,3).traverseU(i => Const(List(i))).getConst == List(1,2,3).foldMap(List(_))
+      List(1,2,3).traverse(i => Const.of[List[Int]](List(i))).getConst == List(1,2,3).foldMap(List(_))
```

Comment:
for some reason the `traverse` version can no longer infer the type correctly. Thus I had to pass the type into `Const`'s apply method. For convenience, I added an `of`  factory method to `Const`. Please shout out if you know a better solution.